test/functional/unit-sdam/monitoring.test.js

```diff
@@ -22,7 +22,8 @@ describe('monitoring', function () {
     return mock.createServer().then(server => (mockServer = server));
   });
 
-  it('should record roundTripTime', function (done) {
+  // Fails on MacOS on Evergreen (NODE-3713)
+  it.skip('should record roundTripTime', function (done) {
```

Comment:
Needs unskip ticket

Reply:
Added todo comment to https://jira.mongodb.org/browse/NODE-3819